static/src/js/components/ManageCollectionAssociation/__tests__/ManageCollectionAssociation.test.jsx

```diff
@@ -335,55 +332,55 @@ describe('ManageCollectionAssociation', () => {
     })
   })
 
-  describe('when the conceptType is a variable', () => {
-    test('should render variable table', async () => {
-      setup({
-        overrideInitialEntries: ['/variables/V1200000104-SEDAC/collection-association'],
-        overridePaths: 'variables/:conceptId/collection-association',
-        overrideMocks: [variableRecord]
-      })
+  // Describe('when the conceptType is a variable', () => {
```

Comment:
Prefer `.skip` over commenting the tests out. I assume this is not longer a feature and should just be deleted?

Reply:
This can be removed, forgot to removed the commented test